webapp/src/components/sidebar/sidebarBoardItem.tsx

```diff
@@ -160,16 +160,16 @@ const SidebarBoardItem = React.memo((props: Props) => {
                         />
                     </Menu>
                 </MenuWrapper>
-            </div>
+            </button>
             {!collapsed && boardViews.length === 0 &&
                 <div className='octo-sidebar-item subitem no-views'>
                     <FormattedMessage
                         id='Sidebar.no-views-in-board'
                         defaultMessage='No pages inside'
                     />
-                </div>}
+                </button>}
```

Comment:
This is wrong, you are closing a button, but opening a div. I guess it was as part of reverting the last change.